src/stories/components/BigNumber/BigNumber.test.tsx

```diff
@@ -0,0 +1,224 @@
+		});
+	});
+	describe('BigNumber error states', () => {
+		test('UndefinedData renders appropriate warning icon and text', async () => {
+			render(<UndefinedData {...UndefinedData.args} />);
+			const errorDescription = await screen.findByText('No data found');
+			expect(errorDescription).toBeInTheDocument();
+		});
+	});
+	describe('BigNumber using numberFormat', () => {
+		test('CurrencyFormat renders with formatted value', async () => {
+			render(<CurrencyFormat {...CurrencyFormat.args} />);
+			const formattedValue = await screen.findByText('255,56 €');
+			expect(formattedValue).toBeInTheDocument();
+		});
+
+		test('PercentageFormat renders with formatted value', async () => {
+			render(<PercentageFormat {...PercentageFormat.args} />);
+			const formattedValue = await screen.findByText('25%');
+			expect(formattedValue).toBeInTheDocument();
+		});
+
+		test('CompactFormat renders with formatted value', async () => {
+			render(<CompactFormat {...CompactFormat.args} />);
+			const formattedValue = await screen.findByText('12.3B');
+			expect(formattedValue).toBeInTheDocument();
+		});
+	});
+
+	describe('Chart with BigNumber children', () => {
+		test('Chart with BigNumber and Line as children should throw error', () => {
+			expect(() =>
+				render(
+					<Chart data={simpleSparklineData}>
+						<BigNumber dataKey="y" orientation="horizontal" label="Empty">
+							<div></div>
+						</BigNumber>
+						<Line />
+					</Chart>
+				)
+			).toThrow(
+				'If passing BigNumber to Chart only the BigNumber will be displayed. All other elements will be ignored'
+			);
+		});
+
+		test('Chart with multiple BigNumbers only displays first', async () => {
+			render(
+				<Chart data={data}>
+					<BigNumber dataKey="y" orientation="horizontal" label="test" icon={
+						<Icon data-testid="first-icon">
+							<svg></svg>
+						</Icon>
+					}/>
+					<BigNumber dataKey="y" orientation="horizontal" label="test" icon={
+						<Icon data-testid="second-icon">
+							<svg></svg>
+						</Icon>
+					}/>
+					<BigNumber dataKey="y" orientation="horizontal" label="test" icon={
+						<Icon data-testid="third-icon">
+							<svg></svg>
+						</Icon>
+					}/>
+				</Chart>
+			);
+
+			const bigNumber = screen.queryByTestId('first-icon');
+			expect(bigNumber).toBeInTheDocument();
+
+			const secondBigNumber = screen.queryByTestId('second-icon');
+			const thirdBigNumber = screen.queryByTestId('third-icon');
+
+			expect(secondBigNumber).toBe(null);
+			expect(thirdBigNumber).toBe(null);
+		});
+	});
+
+	describe('Sparkline with orientation', () => {
+		test('Vertical orientation sparkline', async () => {
+			render(<SparklineVertical {...SparklineVertical.args} />);
+			const value = await screen.findByText('20');
+			expect(value).toBeInTheDocument();
+
+			const label = await screen.findByText('Visitors');
+			expect(label).toBeInTheDocument();
+
+			const chart = await findChart();
+			expect(chart).toBeInTheDocument();
```

Comment:
We should add an assertion for the line as well so we can be sure that it's being included in the chart.